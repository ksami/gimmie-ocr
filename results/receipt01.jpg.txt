Actual | Predicted | OCR Conf
I EL  A     9. 1- . 7     3   | (NSOY FR S/MILK A, 86) | 31.476196
3E3 MC   1E6     .7 - | (SENGCHOON EGG 55 G, 32) | 56.501167
SEE T MC XI 4 CL111 3 F .3. S | (GD S/V ENR WHT 600G, 86) | 41.399887
SL212 . H. . .1. HJJQ 3-3.- H H.     . | (YOPLAIT YOGURT 1, 86) | 44.490337
HIS-Q 5M -. DRY .I H   A     | (NSOY FR S/MILK A, 86) | 42.484520
5 HES 35 VIM-1 J1 V   22 .- WK - | (GD S/V ENR WHT 600G, 86) | 41.641460
-1-.--    HIARLIL-1 Q   | (CHEF CHINA GARLIC, 48) | 29.883698
1 3 PEPQER 1 .L | (RED PEPPER, 54) | 34.803421
LWHEF   1 -.     .963 | (CHEF LG ONION RED, 32) | 36.873001
. LLI U 7 . WK. | (LINK CARD, 40) | 60.441303
BASIL 31.28 NILN RED     T | (RED PEPPER, 86) | 47.076832
  X   | (FAIRPRICE XTRA, 60) | 28.651390
W W W | (GD S/V ENR WHT 600G, 86) | 49.480515
P -- 22 4536 | (P HERB IT/PARSLY30G, 86) | 69.200706
13 IT/PARSLYSOG 4.95. | (P HERB IT/PARSLY30G, 59) | 59.113277
M 5  . | (SENGCHOON EGG 55 G, 60) | 45.421181
T3   SB | (P HERB IT/PARSLY30G, 51) | 61.157806
-  I 3..  -   | (BASIL 30G, 60) | 46.405083
H 3 . Q5 | (YOPLT L/YOG B/B 125G, 45) | 65.248390
X 1 W. | (YOPLAIT YOGURT 1, 86) | 48.205284
.  1 H. . 2H 9...... . | (GD S/V ENR WHT 600G, 43) | 57.812401
WWW AL    -  L | (TOTAL, 36) | 50.739742
