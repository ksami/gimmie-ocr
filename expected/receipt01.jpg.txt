FAIRPRICE XTRA
NEX HYPERMART
!23 Serangoon Central #03-42
!Singapore 556083
!TEL: 6634 4947/4948
!REG No: RCS 1.1.191.1
!GST No: M4-004578-0
SGD
SAN.R VEGR SPR375 1.95
NORMAL PRICE 2.50
CHEF CHINA GARLIC 1.80
RED PEPPER 1.30
CHEF LG ONION RED 1.10
BASIL 30G 2.99
TOMATO 1.61
RAD SHRED-MOZZ 250G 4.95
P HERB IT/PARSLY30G 3.99
SUBTOTAL 19.69
TOTAL 19.69